.screepsrc.example

```diff
@@ -0,0 +1,7 @@
+;If you launch the server without running the local Steam client, 
+;then the Steam Web API key is required for authenticating users. 
+;It can be obtained on this page: http://steamcommunity.com/dev/apikey
+steam_api_key = SOMEKEY
+
+;The server password which should be provided on user sign in.
+password = somepassword
```

Comment:
Maybe add, that this file needs to be copied to the `test-private-server` directory